requirements-dev.txt

```diff
@@ -10,7 +10,6 @@
 # scripts/convert-requirements-to-conda-yml.py as though it can only be found
 # on pip.
 
-Cython<3.0.0
```

Comment:
Cython doesn't really belong in requirements because it's only needed to build invest, and because it's listed under `build-system.requires` in `pyproject.toml`, pip installs it for us in a separate virtual environment at build time.